notebooks/official/migration/sdk-automl-text-entity-extraction-batch-prediction.ipynb

```diff
@@ -23,6 +23,16 @@
         "# limitations under the License."
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "90a6064798e9"
+      },
+      "source": [
+        "Starting on September 15, 2024, you can only customize classification, entity extraction, and sentiment analysis models by moving to Vertex AI Gemini prompts and tuning. Training or updating models for Vertex AI AutoML for Text classification, entity extraction, and sentiment analysis objectives will no longer be available. You can continue using existing Vertex AI AutoML Text objectives until June 15, 2025. For more information about how Gemini offers enhanced user experience through improved prompting capabilities, see \n",
+        "[Overview of model tuning for Gemini](https://cloud.google.com/vertex-ai/generative-ai/docs/models/tune-gemini-overview)"
```

Comment:
The title of the page was recently changed. The link text should be:
 
- "Introduction to tuning"